Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -186,7 +186,8 @@ public final class RootTabBarViewController: UITabBarController {
   private func viewControllerAndParam<T, P>(with index: RootViewControllerIndex, param: P) -> (T, P)? {
     guard
       let vcs = self.viewControllers,
-      let vc = vcs[clamp(0, vcs.count - 1)(index)] as? T
+      let nav = vcs[clamp(0, vcs.count - 1)(index)] as? UINavigationController,
```

Comment:
The `self.viewControllers` is actually an array of `UINavigationController`. Now we first try to cast to UINavigationController and then cast the viewController to whichever type it is. 
This was breaking the navigation to the dashboard when trying to navigate to it through deeplink.